content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -192,7 +192,7 @@ version.
    baseurl=https://pkgs.k8s.io/core:/stable:/v{{< param "version" >}}/rpm/
```

Comment:
This one also needs removing.
```suggestion
   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
```

Reply:
oops! Forgot this. THanks for review. Updating now.